extensions/scheduler/runtime/src/main/java/io/quarkus/scheduler/runtime/SchedulerRuntimeConfig.java

```diff
@@ -13,4 +15,9 @@
     @ConfigItem(defaultValue = "true")
     public boolean enabled;
 
-}
\ No newline at end of file
+    /**
+     * Scheduled task will be flagged as overdue if next execution time is exceeded by this period.
```

Comment:
And the unit is second, or?

Reply:
well, it's a Duration, so the unit is defined by the user

Reply:
Well, a user does not use the config class directly. Very often the `application.properties` file is used. 

You're right. The processing of a duration format is standardized in quarkus (and mentioned in any config reference guide).